### PR TITLE
bridgx optimization

### PR DESCRIPTION
**Pull request by lubingGalaxy**
Pull Request Check List:
- [yes] rebase origin/dev branch (**passed**)
- [yes] make format & make build  (**passed**)
- [yes] union test or test in dev env(**passed**)

Detail in branch:
- [ ]
- [ ]

### CODE REVIEW

File: internal/model/network.go [hunk 1]
```diff
@@ -296,42 +325,116 @@ func AddSecurityGroupRule(ctx context.Context, r SecurityGroupRule) error {
...
+		Scan(&oldVpcIds).Error; err != nil {
+		return err
+	}
+	vpcIds := make([]string, 0, len(vpcs))
+	for _, v := range vpcs {
+		vpcIds = append(vpcIds, v.VpcId)
+	}
+	vpcIdDiff := utils.StringSliceDiff(oldVpcIds, vpcIds)
+	if len(vpcIdDiff) > 0 {
+		logs.Logger.Debugf("%s, vpcIdDiff %v", ak, vpcIdDiff)
```
Comment (galaxy-future-liubo) [inline internal/model/network.go:344]: rm debug log

File: internal/model/network.go [hunk 2]
```diff
@@ -296,42 +325,116 @@ func AddSecurityGroupRule(ctx context.Context, r SecurityGroupRule) error {
...
+		Scan(&oldSwitchIds).Error; err != nil {
+		return err
+	}
+	switchIds := make([]string, 0, len(switches))
+	for _, v := range switches {
+		switchIds = append(switchIds, v.SwitchId)
+	}
+	switchIdDiff := utils.StringSliceDiff(oldSwitchIds, switchIds)
+	if len(switchIdDiff) > 0 {
+		logs.Logger.Debugf("switchIdDiff %v", switchIdDiff)
```
Comment (galaxy-future-liubo) [inline internal/model/network.go:372]: rm debug log

File: internal/model/network.go [hunk 3]
```diff
@@ -296,42 +325,116 @@ func AddSecurityGroupRule(ctx context.Context, r SecurityGroupRule) error {
...
+		Scan(&oldSecGrpIds).Error; err != nil {
+		return err
+	}
+	secGrpIds := make([]string, 0, len(groups))
+	for _, v := range groups {
+		secGrpIds = append(secGrpIds, v.SecurityGroupId)
+	}
+	secGrpIdDiff := utils.StringSliceDiff(oldSecGrpIds, secGrpIds)
+	if len(secGrpIdDiff) > 0 {
+		logs.Logger.Debugf("%s, secGrpIdDiff %v", ak, secGrpIdDiff)
```
Comment (galaxy-future-liubo) [inline internal/model/network.go:400]: rm debug log

File: internal/service/cloud.go [hunk 4]
```diff
@@ -51,15 +51,18 @@ func ExpandInDeed(c *types.ClusterInfo, num int, taskId int64) ([]string, error)
...
+func RepairCluster(c *types.ClusterInfo, taskId int64, availableIds []string, allIds []string) int {
+	successNum := len(availableIds)
 	tags := []cloud.Tag{{
 		Key:   cloud.TaskId,
 		Value: strconv.FormatInt(taskId, 10),
 	}}
 	cloudInstances, err := GetInstanceByTag(c, tags)
 	if err != nil {
-		return
+		logs.Logger.Infof("[RepairCluster] GetInstanceByTag failed %v", err)
```
Comment (galaxy-future-liubo) [inline internal/service/cloud.go:62]: logs.logger.Errorf

File: pkg/cloud/alibaba/api.go [hunk 5]
```diff
@@ -728,25 +728,33 @@ func (p *AlibabaCloud) DescribeGroupRules(req cloud.DescribeGroupRulesRequest) (
...
 				Direction:       _secGrpRuleDirection[*rule.Direction],
 				GroupId:         otherGroupId,
 				CidrIp:          cidrIp,
 				PrefixListId:    prefixListId,
 				CreateAt:        *rule.CreateTime,
 			})
 		}
-
 	}
 	if err != nil {
```
Comment (galaxy-future-liubo) [inline pkg/cloud/alibaba/api.go:759]: err 已在 720 行返回。此处无需再判断。

**Review by galaxy-future-liubo — APPROVED** [review]

**Review by liqunn — APPROVED** [review]

**Pull request merged**